test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Did you maybe commit this file by mistake?

Reply:
Yes